lib/grinder_sdk.dart

```diff
@@ -33,6 +33,51 @@ Directory getSdkDir([List<String> cliArgs]) => cli_util.getSdkDir(cliArgs);
 
 File get dartVM => joinFile(sdkDir, ['bin', _sdkBin('dart')]);
 
+/// Utility tasks for for getting information about the Dart SDK and invoking
+/// dart applications.
+class Dart {
```

Comment:
I chose `Dart` since it wraps the `dart` executable, similar to how `Pub` wraps `pub` and `Dart2js` wraps `dart2js`.  However we do have `Analyzer` for `dartanalyzer, so we could do `Dart2js` -> `Compiler` and `Dart` -> `Vm`, but I kind of think the `Dart` prefix is nice.